jpype/__init__.py

```diff
@@ -47,8 +47,7 @@
 __all__.extend(_jcustomizer.__all__)
 __all__.extend(_gui.__all__)
 
-__version_info__ = (0, 7, 4)
-__version__ = ".".join(str(i) for i in __version_info__)
+__version__ = "0.7.6_dev0"
```

Comment:
I would not delete __version_info__ without deprecation. We can easily set it like this:

__version_info__ = tuple(__version__.split('.'))

Reply:
Okay I will update the rerelease with this change.  The hard part is the fact that tuples elements are numbers.   Not sure what to do about it  But I am fine with ("0","7","6_dev0").

Reply:
I would not consider it a fact, but a recommendation. The splitting could be better though.